mteb/tasks/Retrieval/kor/AutoRAGRetrieval.py

```diff
@@ -19,25 +19,25 @@ class AutoRAGRetrieval(AbsTaskRetrieval):
         eval_splits=["test"],
         eval_langs=["kor-Hang"],
         main_score="ndcg_at_10",
-        date=None,
-        form=None,
-        domains=None,
-        task_subtypes=None,
-        license=None,
-        socioeconomic_status=None,
-        annotations_creators=None,
-        dialect=None,
-        text_creation=None,
-        bibtex_citation="""@misc{AutoRAG,
-  author = "Dongkyu Kim and Byoungwook Kim and Donggeon Han",
-  title = "AutoRAG",
-  year = {2024},
-  url = {https://github.com/Marker-Inc-Korea/AutoRAG}
+        date=("2024-08-03", "2024-08-03"),
+        domains=["Government", "Medical", "Legal", "Social"],
+        task_subtypes=["Article retrieval"],
+        license="not specified",
+        annotations_creators="human-annotated",
+        dialect=[],
+        text_creation="created",
+        bibtex_citation="""@misc{kim2024autoragautomatedframeworkoptimization,
+      title={AutoRAG: Automated Framework for optimization of Retrieval Augmented Generation Pipeline}, 
+      author={Dongkyu Kim and Byoungwook Kim and Donggeon Han and Matouš Eibich},
+      year={2024},
+      eprint={2410.20878},
+      archivePrefix={arXiv},
+      primaryClass={cs.CL},
+      url={https://arxiv.org/abs/2410.20878}, 
 }""",
```

Comment:
I think I've filled all the metadata, but I still get the following error.

========================================== short test summary info ==========================================
FAILED tests/test_evaluators/test_RerankingEvaluator.py::TestRerankingEvaluator::test_nAUC - assert 0.9406528189910982 == 0.8694 ± 8.7e-05
FAILED tests/test_evaluators/test_RetrievalEvaluator.py::TestRetrievalEvaluator::test_nAUC[False-expected_naucs1] - assert 0.950708977119359 == 0.8368244286523474 ± 8.4e-05
FAILED tests/test_TaskMetadata.py::test_all_metadata_is_filled_and_valid - ValueError: The metadata of the following datasets is not filled: ['AutoRAGRetrieval']
=================== 3 failed, 957 passed, 244 skipped, 134 warnings in 188.70s (0:03:08) ====================
make: *** [Makefile:23: test] Error 1

Can I please get some help? Thank you.